bodhi/server/models.py

```diff
@@ -1149,6 +1153,10 @@ class Update(Base):
 
     user_id = Column(Integer, ForeignKey('users.id'))
 
+    # Greenwave
+    greenwave_policies_satisfied = Column(Boolean, default=True, nullable=False)
```

Comment:
Rather than keeping a boolean on whether the greenwave policies are satisfied, why not take advantage of the more rich states available with the ```ci_status``` field? That way, we can express more states than just "yes nor no". We have many states like "waiting", "failed", "skipped", "passed", "running", and maybe others I'm forgetting.

I propose that we just use that existing field and allow Greenwave to be the source of information we use to set that state.